From 65bfb7f55d6bb5f58c3225686b4247d367f0c4aa Mon Sep 17 00:00:00 2001
From: Viktor Hofer <viktor.hofer@microsoft.com>
Date: Wed, 15 Jun 2022 10:58:21 +0200
Subject: [PATCH] Eliminate prebuilts

* Update SourceLInk reference to pick up the source-build version
* Update CodeAnalysis references to pick up the source-build versions
---
 src/Jab/Jab.Common.props | 2 +-
 src/Jab/Jab.csproj       | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/Jab/Jab.Common.props b/src/Jab/Jab.Common.props
index 603de67..11c23fc 100644
--- a/src/Jab/Jab.Common.props
+++ b/src/Jab/Jab.Common.props
@@ -12,7 +12,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" PrivateAssets="All" />
+        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="$(MicrosoftCodeAnalysisAnalyzersVersion)" PrivateAssets="All" />
     </ItemGroup>
 
     <ItemGroup>
diff --git a/src/Jab/Jab.csproj b/src/Jab/Jab.csproj
index fc9b84f..bb9dff8 100644
--- a/src/Jab/Jab.csproj
+++ b/src/Jab/Jab.csproj
@@ -25,7 +25,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" />
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpVersion)" />
     </ItemGroup>
 
     <ItemGroup>
@@ -36,7 +36,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkVersion)" PrivateAssets="All" />
     </ItemGroup>
     
     <Import Project="Jab.Common.props" />

